content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md

```diff
@@ -184,21 +214,21 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
 }
 ```
 
-The `drivers` Cordform parameter in the `node` entry lists paths of the files to be copied to the `drivers` sub-directory of the node.
-To copy the same file to all nodes, define `ext.drivers` in the top level, and reuse it for each node by setting `drivers=ext.drivers`.
+The `drivers` `cordform` parameter in the `node` entry lists paths of the files to be copied to the `drivers` sub-directory of the node.
```

Comment:
Deja vu?

Say "The `drivers` parameter" rather than "The `drivers` `cordform` parameter".

Reply:
updated